Specs/Choice/Swedish/BooleanModel.json

```diff
@@ -0,0 +1,506 @@
+[
+  {
+    "Input": "sant",
+    "NotSupportedByDesign": "python,java",
```

Comment:
As currently only Swedish is implemented, you can add a separate "NotSupported" tag for "javascript". The build will fail as JS support doesn't exist yet.
"NotSupportted" indicates that support will come later. "NotSupportedByDesign" indicates that either it won't be implemented, or that there are no current plans to do so.

Reply:
From my part I'll not add support for python, java or even JS in the forseeable future. Should I change them all to "NotSupported" in case someone else feels inclined to contribute? Or maybe put JS to NotSupported and the rest as NotSupportedByDesign? I don't know.

Reply:
I'll change as per your suggestion.